Libraries/Text/Text/RCTTextView.m

```diff
@@ -273,6 +275,32 @@ - (void)didMoveToWindow
 
 #pragma mark - Accessibility
 
+#if TARGET_OS_OSX // TODO(macOS ISS#2323203)
+
+// This code is here to cover for a mismatch in the what Labels and Values mean in RN vs in Text components on MacOS
+// In MacOS a text element will always read its Value, but will only read it's Label if it's value is set
+// In RN (iOS & Android) a text element will only read it's Value if it has no Label, and will always read its Label
+// This code replicates the expected RN behavior in MacOS by: 
+// 1) Setting the Value = the RN Label if one exists and setting it equal to the text's contents otherwise.
+// 2) Making sure that its Label is always nil, so that it doesn't read out the label twice.
+
+- (void)setAccessibilityLabel:(NSString *)label
+{
+  _accessibilityLabel = label;
```

Comment:
should probably be _accessibilityLabel = [label copy] in case a mutable string is sent in